tests/Unit/HtmlProcessor/AbstractHtmlProcessorTest.php

```diff
@@ -860,7 +860,7 @@ public function provideContentTypeMetaTag(): array
     }
 
     /**
-     * @return string[][]
+     * @return array<array-key, array<int, string>>
```

Comment:
For both changes, we can be even more specific and set `string` (instead of `array-key`) for the keys of the outer array.

Reply:
Actually we can be even more specific, since the inner arrays always have the same number of elements.